Ontology/Willow/Capability/Parameter/Limit/Quantity/Pressure/DischargeAirStaticPressureSetpointHighLimit.json

```diff
@@ -0,0 +1,17 @@
+{
+  "@id": "dtmi:com:willowinc:DischargeAirStaticPressureSetpointHighLimit;1",
+  "@type": "Interface",
+  "displayName": {
+    "en": "Discharge Air Static Pressure Setpoint High Limit"
+  },
+  "description": {"en": "The high limit associated with discharge (supply) air static pressure"},
+  "extends" : [
+    "dtmi:com:willowinc:DischargeAirStaticPressureSetpoint;1"
```

Comment:
Should this extend a Setpoint or a Limit? If limits are defining the acceptable range for a setpoint, maybe they should inherit from Limit rather thatn Setpoint.

Reply:
Agreed, for ease of use with analytics, I've removed the setpoint extensions